package.json

```diff
@@ -11,8 +11,14 @@
 		"start:ios": "npm run build && npm run concurrently -- \"npm run watch\" \"npm run react-native -- run-ios\"",
 		"start": "npm run react-native -- start",
 		"test": "jest --coverage",
-		"watch": "npm run build -- -w"
-	},
+		"watch": "npm run build -- -w",
+    	"lint": "npm run lint:src && npm run lint:test",
+    	"lint:fix": "npm run lint:src-fix && npm run lint:test-fix",
+    	"lint:src": "node_modules/.bin/tslint \"src/**/*.ts?(x)\" --project ./tslint.json",
+    	"lint:src-fix": "node_modules/.bin/tslint \"src/**/*.ts?(x)\" --fix --project ./tslint.json",
+    	"lint:test": "node_modules/.bin/tslint \"test/**/*.ts?(x)\" --project ./tslint.json",
+    	"lint:test-fix": "node_modules/.bin/tslint \"test/**/*.ts?(x)\" --fix --project ./tslint.json"
```

Comment:
Indentation is wonky here. Also I forgot I screwed up Ease's tslint sorry :(
I only discovered this recently, the `-p` flag is supposed to point to the `tsconfig` file. Not sure how it ever worked in ease tbh, but because we have linting that relies on the compiler enabled, this would result in some warnings being thrown and those linting rules being ignored. Again my bad sorry